app_flutter/lib/service/google_authentication.dart

```diff
@@ -62,12 +78,20 @@ class GoogleSignInService extends ChangeNotifier {
 
   /// Initiate the Google Sign In process.
   Future<void> signIn() async {
-    user = await _googleSignIn.signIn();
-    notifyListeners();
+    try {
+      user = await _googleSignIn.signIn();
+      notifyListeners();
+    } on PlatformException catch (error) {
+      debugPrint(error.message);
+    }
```

Comment:
This can be tested by closing the sign in popup (and waiting a few ms until the script picks it up)